hack/tallyvotes.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import json
+import pprint
+import subprocess
+import sys
+
+if len(sys.argv) < 2:
+    print("Usage tallyvotes.py <PR_URL>")
+    sys.exit(1)
+
+pr_url = sys.argv[1]
+pr_url_parts = pr_url.split('/')
+org_repo = pr_url_parts[3] + "/" + pr_url_parts[4]
+pr_num = pr_url_parts[-1]
+
+maintainers_url = "https://raw.githubusercontent.com/argoproj/argoproj/master/MAINTAINERS.md"
+maintainers = subprocess.check_output("curl --silent "+maintainers_url+" | grep -e '^|.*github.com' | grep Yes | awk -F'[' '{print $2}' | awk -F']' '{print $1}'", shell=True).split()
```

Comment:
Consider using [requests library](https://github.com/psf/requests) that's more readable and easier to work with structured data?

Reply:
I actually intentionally did not use requests (I've used it in the past) because I wanted the script to work without python dependencies. 

Also the data is not well structured -- it is a markdown file, and so I wanted to use tools like `grep` and `awk` to help parse the data

Reply:
Thanks for clarifying. If that’s the goal then sounds good to me.